recipes/impactx/build.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# avoid side-injection of -std=c++14 flag in some toolchains
+if [[ ${CXXFLAGS} == *"-std=c++14"* ]]; then
+    echo "14 -> 17"
+    export CXXFLAGS="${CXXFLAGS} -std=c++17"
+fi
+# Darwin modern C++
+#   https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
+if [[ ${target_platform} =~ osx.* ]]; then
+    export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+# IPO/LTO does only work with certain toolchains
+WarpX_IPO=ON
+if [[ ${target_platform} =~ osx.* ]]; then
+    WarpX_IPO=OFF
+fi
+
+# configure
+cmake \
+    -S ${SRC_DIR} -B build                \
+    -DCMAKE_BUILD_TYPE=RelWithDebInfo     \
+    -DCMAKE_VERBOSE_MAKEFILE=ON           \
+    -DCMAKE_INSTALL_LIBDIR=lib            \
+    -DCMAKE_INSTALL_PREFIX=${PREFIX}      \
+    -DImpactX_COMPUTE=NOACC               \
+    -DImpactX_IPO=${ImpactX_IPO}          \
+    -DImpactX_amrex_branch=22.08          \
+    -DImpactX_pyamrex_branch=c11acfaf08162fe42c5ee07c086d23f2874fa779 \
```

Comment:
Is the license of amrex and pyamrex included in `license_file`?

Reply:
They use the same license.

Reply:
Do they have the same copyright holder as well?

Reply:
Yes, in both cases LBNL/UC